tests/Feature/DesktopAuthTest.php

```
@@ -0,0 +1,235 @@
+        $this->assertGuest();
+
+        $response = $this->withSession(['desktop.locked_user_id' => $user->id])
+            ->post(route('desktop.login.store', $user), [
+                'password' => 'secret-password',
+            ]);
+
+        $this->assertAuthenticatedAs($user);
+        $response
+            ->assertRedirect(RouteServiceProvider::HOME)
+            ->assertSessionMissing('desktop.locked_user_id');
+    }
```

PHPMD (2.15.0)

[error] The method test_desktop_unlock_requires_password_after_locking is not named in camelCase. (undefined)

(CamelCaseMethodName)